tasks/DisableCpuFreqScaling.rb

```diff
@@ -0,0 +1,16 @@
+class DisableCpuFreqScaling
+  include Genesis::Framework::Task
+
+  init do
+    log "Make sure acpi-cpufreq is enabled" 
+    run_cmd('modprobe acpi-cpufreq')
+  end
+
+  run do
+    num_procs = run_cmd('cat /proc/cpuinfo | grep processor | wc -l')
+    (0..(num_procs.to_i - 1)).each do |n|
+      run_cmd("echo 'performance' > /sys/devices/system/cpu/cpu#{n}/cpufreq/scaling_governor")
+    end
+  end
```

Comment:
For verification I can thing of two ways to do this... check that performance is set for each cpu or check the current CPU freq against `/sys/devices/system/cpu/cpu#{n}/cpufreq/max_cpu_freq`. I think i'll just check both unless you have another way you want to verify.